internal/commands/secrets/update.go

```diff
@@ -0,0 +1,58 @@
+package secrets
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+type CommandUpdate struct {
+	inputs updateInputs
+	realmClient realm.Client
+	output secretOutput
+}
+
+func (cmd *CommandUpdate) Inputs() cli.InputResolver {
+	return &cmd.inputs
+}
+
+func (cmd *CommandUpdate) Flags (fs *pflag.FlagSet) {
+	cmd.inputs.Flags(fs)
+	fs.StringVarP(&cmd.inputs.secret, flagSecret, flagSecretShort, "", updateFlagSecretUsage)
+	fs.StringVarP(&cmd.inputs.name, flagName, flagNameShort, "", updateFlagNameUsage)
+	fs.StringVarP(&cmd.inputs.value, flagValue, flagValueShort, "", updateFlagValueUsage)
+}
+
+func (cmd *CommandUpdate) Setup(profile *cli.Profile, ui terminal.UI) error {
+	cmd.realmClient = profile.RealmAuthClient()
+	return nil
+}
+
+func (cmd *CommandUpdate) Handler(profile *cli.Profile, ui terminal.UI) error {
+	app, err := cli.ResolveApp(ui, cmd.realmClient, cmd.inputs.Filter())
+	if err != nil {
+		return err
+	}
+
+	secretList, err := cmd.realmClient.Secrets(app.GroupID, app.ID)
+	if err != nil {
+		return err
+	}
+
+	toUpdate, err := cmd.inputs.resolveSecret(ui, secretList)
+	if err != nil {
+		return err
+	}
+
+	err = cmd.realmClient.UpdateSecret(app.GroupID, app.ID, toUpdate.ID, cmd.inputs.name, cmd.inputs.value)
+	cmd.output = secretOutput{secret: toUpdate, err: err}
+	return nil
+}
+
+func (cmd *CommandUpdate) Feedback(profile *cli.Profile, ui terminal.UI) error {
+	if cmd.output.err != nil {
+		return ui.Print(terminal.NewTextLog("There was an issue updating the secret: %s", cmd.output.err.Error()))
+	}
+	return ui.Print(terminal.NewTextLog("Successfully updated the secret with the name %s", cmd.output.secret.Name))
```

Comment:
[nit] `"Successfully updated secret" (doesn't need much more in the context of running that whole command)